examples/pyolite - images.ipynb

```diff
@@ -0,0 +1,97 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "541d1dcb-a8eb-4971-8ded-33543e91fea6",
+   "metadata": {},
+   "source": [
+    "# `PIL.Image` Demo\n",
+    "\n",
+    "PIL is the **Python Image Library**, and has excellent [documentation](https://pillow.readthedocs.io/en/latest/index.html).\n",
+    "\n",
+    "This is a simple demonstration of rendering PIL images in a `jupyterlite` notebook. Most of PIL should work as designed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "455bb1da-8f81-4863-8495-1de97abd1446",
+   "metadata": {},
+   "source": [
+    "## Import the Library\n",
+    "\n",
+    "`jupyterlite` comes with PIL, so you only need import what you need from it."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c84e7d32-aae0-4fd1-8609-a3509b2113f2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from PIL import Image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "00083e52-68d6-461f-8007-5804f57a1f21",
+   "metadata": {},
+   "source": [
+    "## Create a new Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a50cc2a-e53b-4687-83f5-b353fffd6721",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "image = Image.new(\"RGB\", (100, 100))\n",
+    "image"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9463b853-df3e-46ca-9018-21af1c55067e",
+   "metadata": {},
+   "source": [
+    "## Open an existing Image\n",
+    "\n",
+    "Requires that the image be included in `jupyterlite`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "115e688f-9a77-422b-9801-485ba4c7401b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logo = Image.open(\"jupyter.png\")\n",
```

Comment:
Maybe we can remove that once since reading from existing files from Python is not supported yet?

Or we generate / write an image to the Python file system, and then open it with PIL?

Reply:
I'll make loading a file work; I just didn't know what was wrong :) So if I save an image, and then load it, it should work?

Reply:
or use `await js.fetch`... a `py - js.ipynb` that collects tricks for that would be rad... the `micropip` implementation does some [pithy stuff for working with binary downloads](https://github.com/pyodide/pyodide/blob/main/packages/micropip/micropip/micropip.py#L38).

We should probably have a notebook just for js-in-py-js tricks like this until we figure out how to e.g. make stdlib (or `requests`, `httpx`, etc) work.

Reply:
>  So if I save an image, and then load it, it should work?

I think it should yes